src/components/FormSelectionContainer/FormSelect/FormSelect.tsx

```diff
@@ -134,15 +134,15 @@ export const FormSelect: FunctionComponent<FormSelectProps> = ({ id, form, onAdd
           className={`bg-white w-11/12 h-full p-4 leading-5 ${
             formStatus === FormStatus.PENDING || formStatus === FormStatus.ERROR
               ? "text-cloud-300 bg-cloud-100"
-              : "text-cerulean hover:bg-cloud-100"
+              : "text-cerulean-500 hover:text-cerulean-800 hover:bg-cloud-100"
           }`}
           onClick={() => handleForm()}
           {...props}
         >
           {formStatus === FormStatus.PENDING ? (
             <div className="flex flex-col flex-wrap">
               <div>{form.name}</div>
-              <LoaderSpinner className="content-center self-center mt-1" />
+              <LoaderSpinner className="content-center self-center mt-1" primary="#2D5FAA" />
```

Comment:
`primary` perhaps to put in `constants` folder and export from there

Reply:
ok